atomicapp/utils.py

```diff
@@ -214,3 +215,9 @@ def getRoot():
             return HOST_DIR
         else:
             return "/"
+
+    # generates a unique 12 character UUID
+    @staticmethod
+    def getUniqueUUID():
```

Comment:
I've probably screwed up in expressing myself. I want you to replace my code with this new method so that we have a single way of generation of unique container names. Does it make sense now?